api/configs/feature/__init__.py

```diff
@@ -217,18 +217,18 @@ def CONSOLE_CORS_ALLOW_ORIGINS(self) -> list[str]:
     def WEB_API_CORS_ALLOW_ORIGINS(self) -> list[str]:
         return self.inner_WEB_API_CORS_ALLOW_ORIGINS.split(",")
 
-    HTTP_REQUEST_MAX_CONNECT_TIMEOUT: NonNegativeInt = Field(
-        description="",
+    HTTP_REQUEST_MAX_CONNECT_TIMEOUT: conint(ge=10) = Field(
+        description="connect timeout in seconds for HTTP request, which should be no less than 60",
         default=300,
     )
 
-    HTTP_REQUEST_MAX_READ_TIMEOUT: NonNegativeInt = Field(
-        description="",
+    HTTP_REQUEST_MAX_READ_TIMEOUT: conint(ge=60) = Field(
+        description="read timeout in seconds for HTTP request, which should be no less than 60",
         default=600,
```

Comment:
Considering the http timeouts mainly used for code exectuion via `ssrf`, the current default value `600` is way to high for practice. 

@Yeuoly Could you confirm that `ssrf` service and caller uses this config via `httpx` in the unit of seconds?
@laipz8200 How about lower the default value to 1/10 to 60 seconds?